lib/tasks/acls.rake

```diff
@@ -88,10 +92,12 @@ namespace :acls do
           'permissions' => ['create', 'read', 'update', 'delete']
```

Comment:
change the perms to only read and update, also clear cache before just to be safe

Reply:
this still needs to be addressed